.docker/release/Dockerfile

```diff
@@ -1,11 +1,10 @@
 # moveit/moveit:melodic-release
 # Full debian-based install of MoveIt! using apt-get
 
-FROM moveit/moveit:melodic-ci
+FROM ros:melodic-ros-base
```

Comment:
Why do you want to derive the release container from moveit/moveit:melodic-ci?
Shouldn't it simply extend melodic-ros-base and install MoveIt's release packages?

Reply:
melodic-ci has all the dependencies already installed for moveit, so the overall build time for the release Docker image will be faster both locally on servers. i think it makes more sense to re-use the CI layer, which is the standard i put in place ~3 years ago and has worked well.

Reply:
Actually, melodic-ci has even more dependencies installed (namely all build and test dependencies), while the release container should only install the run dependencies, shouldn't it?
However, I agree that installing all system packages again, will take more time than just downloading the existing image and adding the moveit packages.